app/src/main/java/org/mozilla/focus/exceptions/ExceptionsListFragment.kt

```diff
@@ -73,7 +73,7 @@ open class ExceptionsListFragment : BaseSettingsLikeFragment(), CoroutineScope {
                 return true
             }
 
-            override fun onSwiped(viewHolder: RecyclerView.ViewHolder, direction: Int) {}
```

Comment:
This might actually need adding a dummy implementation ( maybe a single-line comment) because non-abstract methods need a body.

Reply:
Thanks for the heads-up! I'm planning to come back to this PR after https://github.com/mozilla-mobile/reference-browser/pull/2056 lands.